tests/Unit/Api/Request/Search/SearchTest.php

```diff
@@ -0,0 +1,140 @@
+<?php declare(strict_types=1);
+
+namespace PeeHaa\AsyncTwitterTest\Api\Request\Status;
+
+use PeeHaa\AsyncTwitter\Api\Request\Search\Search;
+use PHPUnit\Framework\TestCase;
+
+class SearchTest extends TestCase
+{
+    private $request;
+
+    public function setUp()
+    {
+        $this->request = new Search('Search Text');
+    }
+
+    public function testQueryText()
+    {
+        $this->assertTrue(is_string($this->request->getParameters()[0]->getValue()));
+    }
+
+    public function testGeocodeReturnsSelf()
+    {
+    	$this->assertSame($this->request, $this->request->setGeocode('37.781157', '-122.398720', '1mi'));
+    }
+
+    public function testGeocodeSetsCorrectly()
+    {
+    	$this->request->setGeocode('37.781157', '-122.398720', '1mi');
```

Comment:
Fix spacing (as well as in the rest of the file).

project uses 4 spaces indents